spec/unit/csv_builder_spec.rb

```diff
@@ -192,6 +192,11 @@
     end
     let(:dummy_controller) {
       class DummyController
+
+        def find_collection(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `find_collection(*)` if you want the method to accept any arguments but don't care about them.